gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -215,7 +215,16 @@ The following example shows how to permit the A10 GPU device and A10-24Q vGPU de
 
         $ lspci -nnk -d 10de:
 
-     *Example Output*
+     *Example Output for GPU passthrough*
```

Comment:
@cdesiniotis and @tariq1890 can you help with what the example output should look like there? or let me know how i can generate some on my own, thanks!

Reply:
nvm, i was looking at the wrong samples